tests/hilti/types/tuple/index-not-constant-fail.hlt

```diff
@@ -0,0 +1,13 @@
+# @TEST-EXEC-FAIL: ${HILTIC} -j %INPUT >output 2>&1
+# @TEST-EXEC: btest-diff output
```

Comment:
Btw, I've recently started to add `@TEST-DOC: ...` to new tests with a short one-line summary of what the test is about. Suggest we generally start doing that.